src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -202,18 +203,40 @@ ss::future<> service::configure() {
       _ctx.smp_sg, [has_ephemeral_credentials](service& s) {
           s._has_ephemeral_credentials = has_ephemeral_credentials;
       });
-    co_await _controller->get_security_frontend().local().create_acls(
-      {security::acl_binding{
-        security::resource_pattern{
-          security::resource_type::topic,
-          model::schema_registry_internal_tp.topic,
-          security::pattern_type::literal},
-        security::acl_entry{
-          principal,
-          security::acl_host::wildcard_host(),
-          security::acl_operation::all,
-          security::acl_permission::allow}}},
-      5s);
+
+    std::vector<cluster::errc> err_vec;
+    // This could be any non-success errc
+    err_vec.push_back(cluster::errc::timeout);
+    auto create_acls_backoff = initial_create_acls_backoff;
+
+    while (err_vec[0] != cluster::errc::success) {
```

Comment:
issue: This loop should probably terminate on some conditions such as a `gate`, `abort_source`, etc, or do we expect one of the functions to throw something to terminate this?

Reply:
A gate or abort source would be good. None of the other methods are expected to throw.